.github/workflows/lint.yaml

```diff
@@ -0,0 +1,31 @@
+name: Lint
+
+on: [push, pull_request, workflow_dispatch]
```

Comment:
Take the same as thing one:
- https://github.com/srcdslab/sm-plugin-CustomChatColors/blob/f2257730bfe6d40cc637ee9e5c79bc12ef15d12d/.github/workflows/ci.yml#L4